src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -335,6 +335,22 @@ FINANCIAL_REPORTS:
     ROOT_PATH: sandbox
     STORAGE_TYPE: localfs
 FOOTER_ORGANIZATION_IMAGE: /images/logo.png
+FORUM_V2_ENABLE_MYSQL_BACKEND: true
+FORUM_SEARCH_BACKEND: "forum.search.es.ElasticsearchBackend"
+
+FORUM_ELASTIC_SEARCH_CONFIG:
+  - host: "{{ keyOrDefault \"elasticsearch/host\" \"elasticsearch.service.consul\" }}"
+    port: "443"
+    use_ssl: true
+
+FORUM_ELASTIC_SEARCH_CONFIG_ES7:
+  - host: "{{ keyOrDefault \"elasticsearch/host\" \"elasticsearch.service.consul\" }}"
+    port: "443"
+    use_ssl: true
```

Comment:
This block doesn't have any bearing on the V2 forum. That was a setting that was used in the cs_comments_service forum during the migration from a much older ES to the v7.x ES server.
```suggestion
```